pages/book/security-best-practices.mdx

```diff
@@ -0,0 +1,144 @@
+# Security Best Practices
+
+In Tact smart contracts there are various anti-patterns and potential attacks developers need to be cautious of. These can impact the security, efficiency, and correctness of the contracts. Below, are covered both common anti-patterns and attacks specific to Tact smart contracts.
+
+## Send private data on chain
+
+Everything is stored on the blockchain. Remember that even if no private data is saved in the transaction, anyone can still get this data.
```

Comment:
Please, clarify and elaborate. For example, I'd suggest using "privacy sensitive", just "sensitive" or "confidential" over "private", as the latter also implies that something cannot be read and/or is unaccessible.

Reply:
I'd rewrite this paragraph, because `Remember that even if no private data is saved in the transaction, anyone can still get this data.` seems confusing.

You probably meant that the whole smart contract computation is transparent and if you had some confidential values in runtime these could be retrieved with a simple emulation (e.g. https://retracer.ton.org/).